cmd/uzfs_test/uzfs_zvol_zap.c

```diff
@@ -170,7 +178,7 @@ uzfs_zvol_zap_operation(void *arg)
 	end = now + (hrtime_t)(total_time_in_sec * (hrtime_t)(NANOSEC));
 
 	zfs_txg_timeout = 1;
-	ts.tv_sec = 3;
+	ts.tv_sec = 5;
```

Comment:
this kind of timebased ones can again break in future. should we have some other way of verifying them? and also, this comment is not needed to be taken up in this PR.

Reply:
ok.